api/v1beta1/groupversion_info.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright The Kubernetes Authors.
```

Comment:
Let's keep it with kubernetes-nmstate authors?

Reply:
`register.go` is not longer needed

Reply:
The copyring is missing on all the other license headers

Reply:
I will change the places were we have to be mentioned